src/derivation.ts

```diff
@@ -94,27 +106,27 @@ function hasDeriver(pathType: string): pathType is keyof typeof derivers {
  * - A lone BIP-39 path node
  * - A multipath
  *
- * @param pathSegment - The path segment string to validate.
+ * @param path - The path segment string to validate.
```

Comment:
Why are we renaming this?

Reply:
To match the options key?

Reply:
Before we would have `pathSegment`, `pathToChild`, `pathValue`, `pathPart`, etc. depending on which function you are using. I changed it all to `path` for consistency.